package/amazon-alexa/avs-device-sdk/0003-Build-the-SampleApplication-as-library.patch

```diff
@@ -1,177 +0,0 @@
-From dc7e0fcced005d7d4fa9dabafa61e544949c0f08 Mon Sep 17 00:00:00 2001
```

Comment:
Is SampleApp built as a library after removing this patch?

Reply:
Now the support to build SampleApp as library is already available with latest SDK